plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1229,6 +1233,68 @@ var _ = Describe("validator", func() {
 						Expect(err).ToNot(HaveOccurred())
 					})
 				})
+				Context("HA shoot scheduling checks on seed", func() {
+					It("should reject scheduling of multi-zonal HA shoot on non multi-zonal seeds", func() {
+						Expect(coreInformerFactory.Core().InternalVersion().Projects().Informer().GetStore().Add(&project)).To(Succeed())
+						Expect(coreInformerFactory.Core().InternalVersion().CloudProfiles().Informer().GetStore().Add(&cloudProfile)).To(Succeed())
+						Expect(coreInformerFactory.Core().InternalVersion().Seeds().Informer().GetStore().Add(&seed)).To(Succeed())
+						Expect(coreInformerFactory.Core().InternalVersion().SecretBindings().Informer().GetStore().Add(&secretBinding)).To(Succeed())
+
+						shoot.Annotations = make(map[string]string)
+						shoot.ObjectMeta.Annotations[v1beta1constants.ShootAlphaControlPlaneHighAvailability] = v1beta1constants.ShootAlphaControlPlaneHighAvailabilityMultiZone
+						attrs := admission.NewAttributesRecord(&shoot, nil, core.Kind("Shoot").WithVersion("version"), shoot.Namespace, shoot.Name, core.Resource("shoots").WithVersion("version"), "", admission.Create, &metav1.CreateOptions{}, false, nil)
+						err := admissionHandler.Admit(ctx, attrs, nil)
+						Expect(err).To(BeForbiddenError())
+					})
+					It("should reject scheduling of single-zonal HA shoot on non multi-zonal seeds", func() {
```

Comment:
Nit: separate `It()`s with an empty line:
```suggestion

					It("should reject scheduling of single-zonal HA shoot on non multi-zonal seeds", func() {
```